libcrux-ml-dsa/src/simd/avx2/ntt.rs

```diff
@@ -217,6 +437,62 @@ unsafe fn ntt_at_layer_7_and_6(re: &mut AVX2RingElement) {
         mm256_set1_epi32(crate::simd::traits::INVERSE_OF_MODULUS_MOD_MONTGOMERY_R as i32);
 
     #[inline(always)]
+    #[hax_lib::requires({
```

Comment:
The changes in here need some explanation.
Also they appear to break C extraction.

Reply:
fixed. it seems like eurydice does not like the `one_layer` function which was generated by a macro.